pkg/apis/workload/v1alpha1/synctarget_types.go

```diff
@@ -175,6 +174,9 @@ type SyncTargetList struct {
 	Items []SyncTarget `json:"items"`
 }
 
+// ImportedAPISExportName is singleton name of compute service exports in location workspace.
```

Comment:
Is the fact that this is a singleton a new constraint on the topology of a `Location` workspace? If so, can we add this to the documentation?

Reply:
It's mainly a rename of the previous local `kubernetes` APIExport.
But I agree that we should update the documentation.